src/sentry/eventstore/models.py

```diff
@@ -373,21 +358,16 @@ def get_hashes(self, force_config: StrategyConfiguration | None = None) -> Calcu
         from sentry.grouping.api import sort_grouping_variants
 
         variants = self.get_grouping_variants(force_config)
-        flat_variants, hierarchical_variants = sort_grouping_variants(variants)
+        flat_variants = sort_grouping_variants(variants)
```

Comment:
I support not doing it in this PR, which is big enough already, but we could further clean this up by getting rid of the `flat_` business, now that they're the only ones.